location_driver.c

```diff
@@ -122,8 +123,10 @@ void driver_location_stop(void)
 
    // Flush SRAM to disk

```

Comment:
Don't use C++ comments in C files

Reply:
Why of all places is driver_location_stop being picked as a place to save SRAM? This should be moved to somewhere higher-level.

Reply:
made a mistake- fixed comments

any suggestions where to move it to? placed at driver_location_stop as memory is flushed when stopped

Reply:
Can it be placed higher level? Like in the function where driver_location_stop gets called instead?

Reply:
updated as suggested. appreciate the good input.